dbfit-java/core/src/main/java/dbfit/api/DBEnvironment.java

```diff
@@ -35,10 +32,15 @@
             throws SQLException;
 
     /**
-     * This method creates an insert command that will be used to populate new
-     * rows in a table.
+     * Create an insert command that will be used to populate new rows in a table
      */
-    PreparedStatement buildInsertPreparedStatement(String tableName, DbParameterAccessor[] accessors)
+    PreparedDbCommand buildInsertStatement(String tableName, DbParameterAccessor[] accessors)
+            throws SQLException;
+
+    /**
+     * Build a call command for the given stored procedure name and parameters
+     */
+    DbCommand buildStoredProcedureCall(String name, DbParameterAccessor[] accessors)
```

Comment:
I think we may stick to following convention:
* `buildXXX` - building command out of object name, accessors, etc.
* `createXXX` - creating command out of plain text statement.